### PR TITLE
Initial version.

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+version: 2
+jobs:
+  build:
+    machine: true
+    working_directory: ~/eventuate-examples-java-customers-and-orders
+    steps:
+      - checkout
+      - restore_cache:
+          key: eventuate-examples-java-customers-and-orders-{{ checksum "gradle.properties" }}
```
Comment (cer) [inline .circleci/config.yml]: Please fix name

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      - run:
+          command: |
+            export TERM=dumb
+            cd common-contracts
```
Comment (cer) [inline .circleci/config.yml]: ?

Comment (dartartem) [reply .circleci/config.yml]: Sorry, I missed it

File: .circleci/config.yml (hunk 1 above)
Comment (dartartem) [reply .circleci/config.yml]: I see that you already did it

**Pull request merged**